terrautils/formats.py

```diff
@@ -0,0 +1,134 @@
+"""Formats
```

Comment:
would be nice to have a test that takes a trivial array, exports a file in each type, and checks the properties.